packages/react-components/src/organisms/ResearchOutputContributorsCard.tsx

```diff
@@ -78,7 +81,7 @@ const ResearchOutputContributorsCard: React.FC<
     <AuthorSelect
```

Comment:
This field is marked as required however there is no validation to ensure that is been included. A plan needs to be made to address that

Reply:
Created ticket, Tiff told me we will refine it next week.